README.md

```diff
@@ -1,281 +1,133 @@
+* **[Bosh](https://bosh.io/docs/)**
+  for provisioning and managing the virtual machines on which Concourse and CloudFoundry will run.
+* **[Concourse](https://concourse-ci.org/)**
+  for continuous integration and continuous deployment pipelines. The CloudFoundry deployment pipelines will run here.
+* **[AWS VPC](https://aws.amazon.com/vpc/)**
+  for isolating the AWS resources created by `paas-bootstrap` and `paas-cf`.
+* **[AWS RDS databases](https://aws.amazon.com/rds/)**
+  for storing the state of Bosh and Concourse
+* **[Load balancers](https://aws.amazon.com/elasticloadbalancing/) and [TLS certificates](https://aws.amazon.com/certificate-manager/)**
+  for providing secure ingress to Bosh and Concourse
+
+It does not include the AWS IAM roles which are assumed by different system components. Those are created in the account wide terraform (private repository).
+
+## How does the bootstrap work?
+`paas-bootstrap` is designed to solve the bootstrap problem. In our context: how do you deploy the software needed to deploy the rest of the system?
+
+`paas-bootstrap` solves this by deploying a minimal version of Concourse ("Concourse Lite") on to an [AWS EC2](https://aws.amazon.com/ec2/) instance in the default AWS VPC, using [Hashicorp Vagrant](https://www.vagrantup.com/) and the [`vagrant-aws`](https://github.com/mitchellh/vagrant-aws) plugin, and is granted permission to create further AWS resources by the `concourse-lite` instance profile it assumes.
+
+The `create-bosh-concourse` pipeline can be run on from Concourse Lite, which will create the persistent AWS resources, and configure the software running on top of them.
+
+The persistent resources created will include a production configuration of Concourse, with the `create-bosh-concourse` pipeline.
+
+## Deploying a new environment
+`paas-bootstrap` can build the foundations for two types of environments:
+* CloudFoundry
+* Build environments (i.e. docker containers, bosh releases etc)
+
+The full bootstrap process below is only needed the first time an environment is deployed.
+
+These instructions contain placeholders where the exact command may vary. The below table explains the purpose of those placeholders
+
+| Placeholder   | Purpose                                                                                                                                                                                                           |
+| ------------- | ------------------------------------------|
+| `$ACCOUNT`    | The AWS account being targeted (e.g. `dev`, `staging`)|
+| `$ENV` | <p>The name of the environment being targeted. In the case of short lived development environments, this should have a value of `dev`, and the specific environment is set by the `DEPLOY_ENV` environment variable (max 8 chars).</p> <p>Do not use the same `DEPLOY_ENV` value for both types of bootstraps, because that will cause resource allocation conflicts and things will break</p>|
```

Comment:
```suggestion
| `$ENV` | <p>The name of the environment being targeted. In the case of short-lived development environments, this should have a value of `dev`, and the specific environment is set by the `DEPLOY_ENV` environment variable (max 8 chars).</p> <p>Do not use the same `DEPLOY_ENV` value for both types of bootstraps, because that will cause resource allocation conflicts and things will break</p>|
```